Realm/ObjectServerTests/SwiftObjectServerTests.swift

```diff
@@ -765,6 +763,8 @@ class SwiftObjectServerTests: SwiftSyncTestCase {
     }
 
     func testAppInit() {
+        let appName = "translate-utwuv"
```

Comment:
Any particular reason this was made local?

Reply:
This test is the only place it's used so there's no reason for it to have a larger scope.